test/functional/testlib.bash

```diff
@@ -123,11 +123,22 @@ validate_param() {
 # - STREAM: the content to be written
 write_to_protected_file() {
 
-        local arg
-        [ "$1" = "-a" ] && { arg=-a ; shift ; }
-        local file=${1?Missing output file in write_to_protected_file}
-        shift
-        printf "$@" | sudo tee $arg "$file" >/dev/null
+	# If no parameters are received show usage
+	if [ $# -eq 0 ]; then
+		cat <<-EOM
+			Usage:
+			    write_to_protected_file [-a] <file> <stream>
+
+			Options:
+			    -a    If used the text will be appeneded to the file, otherwise it will be overwritten
```

Comment:
typo: appended

Reply:
yeah, maybe I missed something when selecting what should be in this PR, I'll check why tests are failing, thanks for reviewing